### PR TITLE
Remove pittsburgh beta banner for now

### PR DESCRIPTION
**Pull request by kelsonic**
## Description
**Issue:** https://github.com/department-of-veterans-affairs/va.gov-team/issues/3725

**Current Behavior:** The beta banner shows [here](http://localhost:3001/pittsburgh-health-care/).

**Desired Behavior:** The beta banner does not exist [here](http://localhost:3001/pittsburgh-health-care/).

This PR implements the above behavior.

## Testing done
Locally.

## Screenshots
image

## Acceptance criteria
- [x] Remove beta banner for now [here](http://localhost:3001/pittsburgh-health-care/)

## Definition of done
- [x] Events are logged appropriately
- [x] Documentation has been updated, if applicable
- [x] A link has been provided to the originating GitHub issue (or connected to it via ZenHub)
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: src/site/includes/header.html [hunk 1]
```diff
@@ -68,7 +68,7 @@
   {% include "src/site/includes/top-nav.html" %}
 {% endif %}
 
-{% include "src/site/blocks/pittsburgh-beta-banner.liquid" %}
+<!-- {% include "src/site/blocks/pittsburgh-beta-banner.liquid" %} -->
```
Comment (ncksllvn) [inline src/site/includes/header.html]: I think they only want to do this in Staging, correct? Or Prod too?

Comment (kelsonic) [reply src/site/includes/header.html]: Oh good question! Let me ask :)

Comment (kelsonic) [reply src/site/includes/header.html]: Confirmed with @KevinMHoffmanUSDS and it is indeed only staging! Fix incoming! :)

Comment (kelsonic) [reply src/site/includes/header.html]: Updated!

**Review by ncksllvn — APPROVED** [review]

**Comment by kelsonic** [comment]
This is ready for review @department-of-veterans-affairs/frontend-review-group :)

**Review by bkjohnson — APPROVED** [review]

**Pull request merged**